README.md

```diff
@@ -158,3 +164,86 @@ nessus-agent:
   lookup:
     nessus_groups: 'NessusAgents'
 ```
+
+### Logging Configuration
+
+Common to both Windows and Linux are the parameters to configure the logging
+rotation and size.  To apply these settings, add them to the `nessus-agent:lookup`
+pillar dictionary.  These parameters are _optional_ and are not required to be
+configured in the pillar dictionary.
+
+The rotation and size of the logging system can be controlled by setting the following
+paramters:
+
+-   [Rotation Strategy](#nessus-agentlookuplog_rotation_strategy)
+-   [Rotation Time](#nessus-agentlookuplog_rotation_time)
+-   [Max Size](#nessus-agentlookuplog_max_size)
+-   [Max Files](#nessus-agentlookuplog_max_files)
+
+For more information, please refer to vendor documentation:
+
+[How to Manage Nessus log size and rotation](https://community.tenable.com/s/article/How-to-manage-Nessus-log-size-and-rotation)
+
+#### `nessus-agent:lookup:log_rotation_strategy`
+
+The `log_rotation_strategy` parameter can be set to `"daily"` or `"size"`.
+
+**Required**: `False`
+>
+>**Default**: ``
+
+**Example**:
+
+```yaml
+nessus-agent:
+  lookup:
+    log_rotation_strategy: `"daily"`
```

Comment:
The backticks around "daily" are rendered _literally_ inside code blocks... Meaning it will look like the backticks are required in the yaml itself...

Reply:
Okay.  I actually meant it to be single quotes.  This maybe also be related to your earlier comment about the single quotes.  When modifying the `log.json` file, I need it to actually include the text with the double quotes, i.e. `"daily"`.

I'll check and see if the double quotes are rendered properly without the single quotes.